spark/arctern_spark/geoseries.py

```diff
@@ -107,6 +105,38 @@ def _validate_args(*args, dtype=None):
 
 
 class GeoSeries(Series):
+    """
+    One-dimensional Series to store an array of geometry objects.
+
+    Parameters
+    ----------
+    data : array-like, Iterable, dict, or scalar value(str or bytes)
+        Contains geometric data stored in GeoSeries. The geometric data can be in `WKT <https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry>`_ or `WKB <https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry#Well-known_binary>`_ format.
```

Comment:
According to koalas Series doc, add `Note that if `data` is a pandas Series, other arguments should not be used.` will be better.

Reply:
Yeah,  it should be added.